static/app/components/events/interfaces/debugMeta/debugImageDetails/candidate/information/features.tsx

```diff
@@ -29,7 +29,7 @@ function Features({download}: Props) {
 
   return (
     <Fragment>
-      {Object.keys(ImageFeature).map(imageFeature => {
+      {Object.values(ImageFeature).map(imageFeature => {
```

Comment:
convert from keys() to values() because we renamed the keys, but before this change the keys and values we the same.